src/main/resources/META-INF/rewrite/jspecify.yml

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2024 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.jspecify.MigrateToJspecify
+displayName: Migrate to jspecify
+description: >-
+  This recipe will migrate to jspecify annotations.
+tags:
+  - java
+recipeList:
+  - org.openrewrite.java.jspecify.MigrateFromJavaxAnnotationApi
+  - org.openrewrite.java.jspecify.MigrateFromJakartaAnnotationApi
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.jspecify.MigrateFromJavaxAnnotationApi
+displayName: Migrate to JSpecify
+description: Migrate from javax Annotation API to JSpecify.
```

Comment:
Probably best to give each recipe a unique name, otherwise the links in the docs would look the same in a list of recipes.

Reply:
Done